### ISSUE TITLE
[GENERAL] How to lighten up your WoW login screen

### ISSUE THREAD

**Issue by MaelstromServer** [azerothcore/forum#3]
It's useful to have a light WoW screen interface, especially if you are a developer and often open/close many WoW clients a the same time:

http://mangosrumors.co.nf/how-to-lighten-up-your-wow-login-screen/

[![image](https://user-images.githubusercontent.com/48177002/53697172-e1a8bc80-3dce-11e9-9133-fae5e27e22f4.png)](http://mangosrumors.co.nf/how-to-lighten-up-your-wow-login-screen/)

**Comment by BarbzYHOOL** [comment]
I just tried it few days ago and no, it doesn't help me at all (on Linux with wine though).

**Comment by XdevTLKWoW** [comment]
Not seing any substantial difference.But nice work 📦

**Comment by BarbzYHOOL** [comment]
yes, it's still better than that annoying wotlk dragon (especially with multiple clients) so still interesting :P

**Comment by kenpb** [comment]
I used to run the [Marforius Client](https://github.com/Marforius/Marforius-Client), it had some neat perks like saving multiple account credentials and realms, it also allowed to choose the login screen, so you can put the vanilla one too, furthermore, if you're interested in doing bots or the like I also forked a [version](https://github.com/kenpb/Marforius-Client) with unlocked LUA.

**Comment by FrancescoBorzi** [comment]
@kenpb nice one, I suggest to open a separate post to give that more visibility :)

**Comment by kenpb** [comment]
Sure, will do!
